apps/mark/frontend/src/lib/gamepad.ts

```diff
@@ -88,23 +38,23 @@ export function handleGamepadButtonDown(buttonName: Button): void {
 /* istanbul ignore next - triggering keystrokes issue - https://github.com/votingworks/bmd/issues/62 */
 export function handleGamepadKeyboardEvent(event: KeyboardEvent): void {
   switch (event.key) {
-    case 'ArrowUp':
-      handleArrowUp();
+    case Keybinding.FOCUS_PREVIOUS:
+      advanceElementFocus(-1);
       break;
-    case '[':
-    case 'ArrowDown':
-      handleArrowDown();
+    case Keybinding.LEGACY_PAT_MOVE:
+    case Keybinding.FOCUS_NEXT:
+      advanceElementFocus(1);
       break;
-    case 'ArrowLeft':
-      handleArrowLeft();
+    case Keybinding.PAGE_PREVIOUS:
+      triggerPageNavigationButton(PageNavigationButtonId.PREVIOUS);
       break;
-    case 'ArrowRight':
-      handleArrowRight();
+    case Keybinding.PAGE_NEXT:
+      triggerPageNavigationButton(PageNavigationButtonId.NEXT);
       break;
-    case ']':
+    case Keybinding.LEGACY_PAT_SELECT:
```

Comment:
Tracking updating the daemon in https://github.com/votingworks/vxsuite/issues/4742 so we can get rid of the `LEGACY` keybindings later.

(Now that I think about it, we probably don't need to handle PAT input in VxMark because there is no PAT input on those machines, but the comment stands for `mark-scan/.../assistive_technology`)

Reply:
Ah, perfect! Would be great to standardise on the `FOCUS_NEXT` and `SELECT` keybindings - wasn't sure if we had control over the key mapping for the PAT input.

And thanks for the heads-up on VxMark - will drop the legacy bindings altogether if there's no PAT support there anyway